### PR TITLE
Upgrades wrt openshift-pipelines v0.4.0 release

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Add openshift/tektoncd-pipeline v0.4.0 payload to operator repository

Updates v0.4.0 downstream release.yaml with
```yaml
env:
  - name: SYSTEM_NAMESPACE
    valueFrom:
      fieldRef:
        fieldPath: metadata.namespace
```
Add generated openshift-pipelines-operator.v0.4.0.clusterserviceversion.yaml

Update `currentCSV` in openshift-pipelines-operator package

Update scripts/olm_catalog.sh as more than one CSV are present

Comments out 'Installed Resources' in CSV as it was degrading UX in CR status view

Replace operator logo with image uri for new logo

openshift

### CODE REVIEW

File: deploy/olm-catalog/openshift-pipelines-operator/0.4.0/openshift-pipelines-operator.v0.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+  replaces: openshift-pipelines-operator.v0.3.1
+  maintainers:
+  - name: Red Hat, Inc.
+    email: customerservice@redhat.com
+  links:
+  - name: Openshift Tektoncd Pipeline GitHub Repository
+    url: https://github.com/openshift/tektoncd-pipeline
+  - name: OpenShift Pipelines Operator GitHub Repository
+    url: https://github.com/openshift/tektoncd-pipeline-operator
+  icon:
```
Comment (siamaksade) [inline deploy/olm-catalog/openshift-pipelines-operator/0.4.0/openshift-pipelines-operator.v0.4.0.clusterserviceversion.yaml]: Hold off please with the icon. Use the previous one till we get to a better resolution with brand

Comment (nikhil-thomas) [reply deploy/olm-catalog/openshift-pipelines-operator/0.4.0/openshift-pipelines-operator.v0.4.0.clusterserviceversion.yaml]: copy that

**Pull request merged**